### DISCUSSION TITLE
Easiest way to add "video" field

### DISCUSSION THREAD

**Discussion by Jue-Xu** [retorquere/zotero-better-bibtex#1772] [Q&A] [answered]
First of all, thanks for your work! Zotero-better-bibtex is an amazing extension that makes my life much easier. 

Recently, many research presentations/seminars are available on Youtube. I would like to add the Youtube link in my reference through biblatex.  I know I can use "Url" or "Note" field for this purpose, but these two fields are often used in my case. So, I want a "video" field in my .bib file. As far as I know, there is no native "video" field in Zotero, so I put the youtube link in the "Extra" field just like "arXiv".

![image](https://user-images.githubusercontent.com/16731479/112203595-191e7780-8be9-11eb-8f94-e397493940a3.png)

Under my current setting, "eprint", "eprinttype", "archiveprefix" will appear automatically in my .bib file and `video: https://youtu.be/ID` would appear in the annotation field. What I want is a "video" field that contains the Youtube link in my .bib file after I put the youtube link in "Extra" field as in the picture above. I suppose this function can be realized by scripting, but I wonder what the easiest way to realize it. Thanks in advance!

**Comment by retorquere** [comment, marked as answer]
Just add it as `tex.video: https://youtu.be/fKUCsw5MJeQ` to the extra field.

**Reply by Jue-Xu** [reply]
Thx! That works!!
